addons/calendar/static/src/activity/activity_list_popover_item_patch.xml

```diff
@@ -6,5 +6,11 @@
                 <i class="fa fa-calendar"/>
             </button>
         </xpath>
+        <xpath expr="//small[@t-if=&quot;props.activity.state !== 'today'&quot;]" position="after">
```

Comment:
This selector is very prone to bugs and could break easily. You could either add a `name/class` attribute on the parent element.